### PR TITLE
AlgoHeist21: Bucket Sort w/signoff

### PR DESCRIPTION
**Pull request by AnshumanDhiman**
Signed-off-by: AnshumanDhiman <67755381+AnshumanDhiman@users.noreply.github.com>

# HacktoberFest 21

Please mention the PR subject as `AlgoHeist21: Algorithm Name`

## Algorithm you like to add

Bucket Sort

## Programming Language for Algorithm Implementation

C++

## Code file name with extension

bucketSort.cpp

## Did you watched Money Heist

Yes / No

### If Yes, mention your favourite character (optional)

mention the character name here (for example Professor)

## YOU ARE

a student / professional (choose any one)

## GITHUB USERNAME

enter your username here

### We are looking for a name for this repository for next hacktober fest please suggest below (optional)

`_____________`

Thanks for your contributions !!

### CODE REVIEW

**Comment by AnshumanDhiman** [comment]
added a bucket sort algorithm, if its ok please merge and add the hacktoberfest label in it

File: sorting/bucket_sort/bucketSort.cpp [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#include <algorithm>
+#include <iostream>
+#include <vector>
+using namespace std;
+
```
Comment (GauravWalia19) [inline sorting/bucket_sort/bucketSort.cpp:5]: Remove the extra space one is enough

File: sorting/bucket_sort/bucketSort.cpp [hunk 2]
```diff
@@ -0,0 +1,42 @@
+#include <algorithm>
+#include <iostream>
+#include <vector>
+using namespace std;
+ 
+
+void bucketSort(float arr[], int x)
+{
+
```
Comment (GauravWalia19) [inline sorting/bucket_sort/bucketSort.cpp:9]: why do you need 2 \n here

**Review by GauravWalia19 — CHANGES_REQUESTED** [review]
Please do the changes as mentioned and update the readme also

**Pull request merged**